adapters/appnexus/params_test.go

```diff
@@ -53,6 +53,8 @@ var validParams = []string{
 	`{"placement_id":123, "keywords":"foo=bar,foo=baz"}`,
 	`{"placement_id":123, "keywords":{"genre": ["rock", "pop"], "pets": ["dog"]}}`,
 	`{"placement_id":123, "use_pmt_rule": true, "private_sizes": [{"w": 300, "h":250}]}`,
+	`{"placementId":123, "ext_inv_code": "invCode"}`,
+	`{"placementId":123, "external_imp_id": "impId"}`,
```

Comment:
should also add JSON framework test for these new params

Reply:
Thank you, @onkarvhanumante! Updated test case to pass external_imp_id and ext_inv_code in [217336d](https://github.com/prebid/prebid-server/pull/2792/commits/217336db5a0b766a27395bf57ea4a70d1dda59f1). It helped catch a bug